### ISSUE TITLE
Data augmentation

### ISSUE THREAD

**Issue by fransonf** [zju3dv/clean-pvnet#72]
Hello! Is there any online data augmentation done to the renderings or the real images when training PVNet on the tless objects? Or is the network strictly trained on the images found in the formatted dataset (e.g. 1296 real images and 5000 renderings for some objects)?

Also, when running `python run.py --type evaluate --cfg_file configs/tless/tless_01.yaml`, is the network evaluated against a separate test set or against a subset in the training set?

Thanks in advance,
Filip

**Comment by pengsida** [comment]
1. Yes, there is online data augmentation
2. Evaluate the network on a separate test set.

**Comment by fransonf** [comment]
Thanks a lot for answer!
